packages/teams-js/test/public/pages.spec.ts

```diff
@@ -330,6 +294,74 @@ describe('Testing pages module', () => {
             expect(pages.navigateCrossDomain('https://valid.origin.com')).rejects.toEqual(errorNotSupportedOnPlatform);
           });
 
+          it(`pages.navigateCrossDomain should throw error with a bad origin when initialized with ${context}`, async () => {
+            await utils.initializeWithContext(context);
+            const promise = pages.navigateCrossDomain('https://badorigin.com');
+            const navigateCrossDomainMessage = utils.findMessageByFunc('navigateCrossDomain');
+            validateRequestWithoutArguments(navigateCrossDomainMessage, 'navigateCrossDomain');
+
+            utils.respondToMessage(navigateCrossDomainMessage!, false);
```

Comment:
Just to clarify, there is no URL validation happening in the App SDK, it is all happening in the hub sdk and these unit tests are abstracting that away, right?

Reply:
Specifically, does the test behave differently if you pass a valid URL on line 299?

Reply:
Yes, that's the case, there's no validation in the App SDK - Shreyas and I discussed offline. But that being true, do we need so many tests validating the same behavior? (That pages.navigateCrossDomain throws the error response that it receives?)